dashboard/app/helpers/levels_helper.rb

```diff
@@ -20,7 +20,37 @@ def url_from_path(path)
     "#{root_url.chomp('/')}#{path}"
   end
 
+  def set_channel
+    # This only works for logged-in users because the storage_id cookie is set
+    # on '/v3' path for non-logged-in users.
```

Comment:
Right, he fixed the `/v3` path issue.  However if `ChannelsApi` decides to set a cookie we won't propagate it back to the user's browser.  I'll update the comment.